testsuite/libra-fuzzer/src/fuzz_targets/secure_json_rpc_client.rs

```diff
@@ -19,12 +27,14 @@ impl FuzzTargetImpl for SecureJsonRpcSubmitTransaction {
         "Secure JSON RPC submit_transaction() response"
     }
 
-    fn generate(&self, _idx: usize, _gen: &mut ValueGenerator) -> Option<Vec<u8>> {
-        Some(generate_submit_transaction_corpus())
-    }
-
     fn fuzz(&self, data: &[u8]) {
-        fuzz_submit_transaction_response(data);
+        let runner = create_test_runner(data);
+        let strategy = arb_submit_transaction_response();
+
+        let input = get_fuzzer_response(runner, strategy);
+        if let Some(input) = input {
+            let _ = process_submit_transaction_response(input);
+        }
```

Comment:
you should be able to use the new helper instead of this code, see my previous PR as well : o